addons/hw_drivers/tools/wifi.py

```diff
@@ -0,0 +1,278 @@
+"""Module to manage Wi-Fi connections and access point mode using
+NetworkManager and ``nmcli`` tool.
+"""
+
+import logging
+import secrets
+import subprocess
+import time
+from pathlib import Path
+from functools import cache
+
+from .helpers import get_ip, get_mac_address, writable
+
+__all__ = [
```

Comment:
This is the first time I've seen `__all__`, from what I've read it is only needed when you have `import *` - what was the reason for adding it?

Reply:
I can remove it, I added it in case someone, one day, decide to `import *`, it's kind of a "security" to avoid people to use methods they shouldn't. If you think it's useless, let's remove it